layouts.css

```diff
@@ -0,0 +1,7 @@
+img {

```

Comment:
To get things to display beside each other, try floating and setting dimensions for your section tags.